docs/sections/user_guide/cli/drivers/cdeps/run-help.out

```diff
@@ -20,8 +20,7 @@ Optional arguments:
   --graph-file PATH
       Path to Graphviz DOT output [experimental]
   --key-path KEY[.KEY...]
-      Dot-separated path of keys leading through the config to the driver's
-      configuration block
+      Dot-separated path of keys to driver config block
```

Comment:
In several places, both in CLI verbiage and in docstrings, I made argument descriptions more consistent, and in some cases more concise.